packages/server/graphql/mutations/helpers/createJiraTask.ts

```diff
@@ -28,7 +26,7 @@ const createJiraTask = async (
   const bestType = issuetypes.find((type) => type.name === 'Task') || issuetypes[0]
   const payload = {
     summary,
-    description,
+    description: convertMdToAdf(markdown),
```

Comment:
If we skip the conversion to markdown, but instead work with [`contentState.getBlockMap()`](https://draftjs.org/docs/api-reference-content-state#getblockmap) directly, it should be quite simple to convert these few [`ContentBlock` types](https://draftjs.org/docs/api-reference-content-block) to [ADF](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/) directly.